lib/portlayer/storage/volume/vsphere/store.go

```diff
@@ -25,13 +25,26 @@ import (
 	"github.com/vmware/vic/lib/config/executor"
 	"github.com/vmware/vic/lib/constants"
 	"github.com/vmware/vic/lib/portlayer/storage"
+	"github.com/vmware/vic/lib/portlayer/storage/volume"
+	"github.com/vmware/vic/lib/portlayer/storage/vsphere"
 	"github.com/vmware/vic/lib/portlayer/util"
 	"github.com/vmware/vic/pkg/trace"
 	"github.com/vmware/vic/pkg/vsphere/datastore"
 	"github.com/vmware/vic/pkg/vsphere/disk"
 	"github.com/vmware/vic/pkg/vsphere/session"
 )
 
+const (
+	// TODO: this was shared with image store hence the disjoint naming. Should be updated
+	// but migration/upgrade implications are unclear
+	metaDataDir = "imageMetadata"
```

Comment:
it seems like it might be a useful constant. Should not it be moved to lib/constant?